src/sparsezoo/objects/model.py

```diff
@@ -164,6 +164,21 @@ def __init__(
             for res in sorted_files[FileTypes.FRAMEWORK.value]
         ]
 
+        self._checkpoints_folders = (
+            [
+                File(
+                    model_metadata=metadata,
+                    # child_folder_name=metadata.checkpoints,
+                    override_folder_name=override_folder_name,
+                    override_parent_path=override_parent_path,
+                    **res,
+                )
+                for res in sorted_files[FileTypes.CHECKPOINTS.value]
+            ]
+            if sorted_files[FileTypes.CHECKPOINTS.value]
+            else None
```

Comment:
let's fall back on an empty array rather than None so reference code can loop through safely without a truth check on it